backend/frontend/views.py

```diff
@@ -295,6 +298,75 @@ def action_start_card(request, card_id):
     )
 
 
+def user_can_request_review(logged_in_user):
```

Comment:
I think we need a naming convention for these permission functions. It's really not clear what the intention of this code is until I go search for where you used it. 

Can you think of a better naming convention? It's also not clear from the name that a card_id is required here. 

Ideally I should know what a function is supposed to do just by looking at the name.